.vscode/settings.json

```diff
@@ -7,5 +7,8 @@
     "source.removeUnusedImports": "explicit"
   },
   "typescript.tsdk": "node_modules/typescript/lib",
-  "typescript.enablePromptUseWorkspaceTsdk": true
+  "typescript.enablePromptUseWorkspaceTsdk": true,
+  "yaml.schemas": {
```

Comment:
@Piotrk39 What is this? Is this intentionally part of this PR?

For this rest I think everything is okay of this PR!

Reply:
I think it has to do with some installs that happen and I have this changes happening for no reason and they slip through. I'll remove it.